### PR TITLE
Feed fixes

### PR DESCRIPTION
**Pull request by meyerweb**
Addresses #44; may not be complete.

### CODE REVIEW

File: feed-creator.js [hunk 1]
```diff
@@ -33,8 +34,8 @@ module.exports = function (meta, data) {
 
       feed.addItem({
         title: item.title,
-        id: `https://bkardell.com/bcd-watch/${item.file}`,
-        link: `https://bkardell.com/bcd-watch/${item.file}`,
+        id: `https://bcd-watch.igalia.com/${item.file}`,
```
Comment (bkardell) [inline feed-creator.js:37]:
```suggestion
        id: `https://bcd-watch.igalia.com/${data.path}/${item.file}`,
```

File: feed-creator.js [hunk 2]
```diff
@@ -33,8 +34,8 @@ module.exports = function (meta, data) {
...
+        link: `https://bcd-watch.igalia.com/${item.file}`,
```
Comment (bkardell) [inline feed-creator.js:38]:
```suggestion
        link: `https://bcd-watch.igalia.com/${data.path}/${item.file}`,
```

**Comment by bkardell** [comment]
Other than the two suggestions I made, this lgtm - it's definitely a nice subset of #44

File: feed-creator.js (hunk 1 above)
Comment (meyerweb) [reply feed-creator.js:37]: `${data.path}` should be `${directory}` (see lines 19-21).

File: feed-creator.js (hunk 2 above)
Comment (meyerweb) [reply feed-creator.js:38]: `${data.path}` should be `${directory}` (see lines 19-21).

File: feed-creator.js [hunk 3]
```diff
@@ -4,25 +4,26 @@ const fs = require('fs'),
...
       // favicon: 'https://commons.wikimedia.org/wiki/File:Stars_Art_-_FREE_%2850210921903%29.jpg',
       updated: new Date(),
-      copyright: 'All rights reserved 2024, Brian Kardell',
+      copyright: 'All rights reserved, Igalia SL',
       generator: 'awesome mix', // optional, default = 'Feed for Node.js'
       feedLinks: {
-        rss: 'https://bkardell.com/bcd-watch/feed.rss',
-        json: 'https://bkardell.com/bcd-watch/feed.json',
-        atom: 'https://bkardell.com/bcd-watch/feed.atom',
+        rss:  `https://bcd-watch.igalia.com/${directory}/feed.rss`,
```
Comment (bkardell) [inline feed-creator.js:19]:
```suggestion
        rss:  `https://bcd-watch.igalia.com/${data.path}/feed.rss`,
```

File: feed-creator.js [hunk 4]
```diff
@@ -4,25 +4,26 @@ const fs = require('fs'),
...
+        json: `https://bcd-watch.igalia.com/${directory}/feed.json`,
```
Comment (bkardell) [inline feed-creator.js:20]:
```suggestion
        json: `https://bcd-watch.igalia.com/${data.path}/feed.json`,
```

File: feed-creator.js [hunk 5]
```diff
@@ -4,25 +4,26 @@ const fs = require('fs'),
...
+        atom: `https://bcd-watch.igalia.com/${directory}/feed.atom`,
```
Comment (bkardell) [inline feed-creator.js:21]:
```suggestion
        atom: `https://bcd-watch.igalia.com/${data.path}/feed.atom`,
```

File: feed-creator.js (hunk 1 above)
Comment (bkardell) [reply feed-creator.js:37]: no directory is "/out/" whereas this is the particular subdirectory (weekly or weekly-completed) - but you're right, those should change too - I made those suggestions as well

Comment (meyerweb) [reply feed-creator.js:37]: No, see line 7: I made directory a local variable where I filtered the “out” part (plus a little more) out of `data.path` in order to not have the URLs look bonkers, and maybe break.

**Pull request merged**